nodeJS/orms/prisma_orm.md

```diff
@@ -0,0 +1,157 @@
+### Introduction
+
+Tired of writing raw SQL queries in your projects? In this lesson, we'll talk about Object Relational Mappers (ORM for short). ORMs are tools that let you manipulate data in your database and are widely used in the software industry. We'll dive deep into one ORM popular in the Node.js landscape: [Prisma ORM](https://www.prisma.io/).
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- What are ORMs, and why are they used?
+- Introduction to Prisma ORM
+- Features of Prisma
+
+### Challenges with raw SQL'in
```

Comment:
Is `'in` a typo? It reads a little odd.

Reply:
It's intended to be a verb: "SQLing" (I don't think it's a correct word but the learners get the point). I shortened it to "SQL'in" to keep the lesson a tad informal. Will keep it as is unless you (or someone else) has further thoughts.

Reply:
I think its fine: I guessed the intention upon first read, so unless this becomes a major point of confusion I'd keep it

Reply:
I have no strong feelings, just thought I'd point it out 👍

Reply:
Technically should be `SQLin'` if you want to use the apostrophe, as you're omitting letters from the end, and nothing between SQL and `in`

Reply:
🤦 how did I not see that, fixed.